### ISSUE TITLE
The document is broken and need repair.

### ISSUE THREAD

**Issue by ah-its-andy** [qax-os/excelize#1764]
**Description**

**Steps to reproduce the issue:**
Test Codes:
```
func TestCreateExcel(t *testing.T) {
	doc := excelize.NewFile()
	defer func() {
		if err := doc.Close(); err != nil {
			t.Errorf("Close excel file failed: %v", err)
		}
	}()
	active, err := doc.NewSheet("Summary")
	if err != nil {
		t.Errorf("NewSheet: %v", err)
	}
	_, err = doc.NewSheet("DataSet")
	if err != nil {
		t.Errorf("NewSheet: %v", err)
	}
	doc.SetActiveSheet(active)
	doc.SaveAs("/path/to/save/test.xlsx")
}
```

**Describe the results you received:**
Excel says the docuement had been broken and it will try to fix it. 

**Output of `go version`:**
1.21

**Excelize version or commit ID:**

```text
github.com/xuri/excelize/v2 v2.8.0
```

**Environment details (OS, Microsoft Excel™ version, physical, etc.):**
Generated in Ubuntu 21, Opened in Windows 11 ,  Microsoft 365(2311),

**Comment by ah-its-andy** [comment]
[test.xlsx](https://github.com/qax-os/excelize/files/13724527/test.xlsx)

**Comment by xuri** [comment]
Thanks for your issue. If you are reporting a new issue, make sure that we do not have any duplicates already exist. If it does not work with Go 1.21.0, please reference the issues #1465, #1595, #1603, #1608, #1614, #1619, #1620, #1621, #1623, #1633, #1637, #1641, #1642, #1648, #1651, #1652, #1656, #1657, #1660, #1663, #1670, #1686, #1734, and #1752. There are some incompatible changes in the Go 1.21.0 encoding/xml library. I have given feedback to the Go team and created a patch for it (golang/go#61881), and it has been fixed on Go 1.21.1. Please using the Go 1.20.12 and previous Go released version or upgrade to Go 1.21.1 and later. I have added notice on the [README](https://github.com/xuri/excelize#introduction) and [documentation website](https://xuri.me/excelize/en/base/installation.html#install) for this, and I've closed this. If you have any questions, please let me know, and reopen this anytime.

谢谢您的议题，在您创建新的问题之前，请确保不存在任何重复的问题。这与已有问题重复，请参考 #1465, #1595, #1603, #1608, #1614, #1619, #1620, #1621, #1623, #1633, #1637, #1641, #1642, #1648, #1651, #1652, #1656, #1657, #1660, #1663, #1670, #1686, #1734 和 #1752。Go 1.21.0 对 encoding/xml 标准库做了不兼容的更改，我已将该问题反馈至 Go 团队，并提交了修复补丁 golang/go#61881，已在 Go 1.21.1 中修复。请使用 Go 1.20.12 以及更早版本，或升级 1.21.1 及以上版本。我已经为此在[自述文档](https://github.com/xuri/excelize/blob/master/README_zh.md#简介)和[文档网站](https://xuri.me/excelize/zh-hans/base/installation.html#install)上添加了通知，因此我将关闭此问题。如果您有任何问题，请告诉我，并可随时重新打开。

**Comment by xuri** [comment]
谢谢您的议题，在您创建新的议题之前，请确保不存在任何重复的议题。这与已有议题重复，请参考 #1465, #1595, #1603, #1608, #1614, #1619, #1620, #1621, #1623, #1633, #1637, #1641, #1642, #1648, #1651, #1652, #1656, #1657, #1660, #1663, #1670, #1686, #1734 和 #1752。Go 1.21.0 对 encoding/xml 标准库做了不兼容的更改，我已将该议题反馈至 Go 团队，并提交了修复补丁 golang/go#61881，已在 Go 1.21.1 中修复。请使用 Go 1.20.12 以及更早版本，或升级 1.21.1 及以上版本。我已经为此在[自述文档](https://github.com/xuri/excelize/blob/master/README_zh.md#简介)和[文档网站](https://xuri.me/excelize/zh-hans/base/installation.html#install)上添加了通知，因此我将关闭此议题。如果您有任何疑问，请告诉我，并可随时重新打开。

**Comment by xuri** [comment]
Thanks for your issue. If you are reporting a new issue, make sure that we do not have any duplicates already exist. If it does not work with Go 1.21.0, please reference the issues #1465, #1595, #1603, #1608, #1614, #1619, #1620, #1621, #1623, #1633, #1637, #1641, #1642, #1648, #1651, #1652, #1656, #1657, #1660, #1663, #1670, #1686, #1734, and #1752. There are some incompatible changes in the Go 1.21.0 encoding/xml library. I have given feedback to the Go team and created a patch for it (golang/go#61881), and it has been fixed on Go 1.21.1. Please using the Go 1.20.12 and previous Go released version or upgrade to Go 1.21.1 and later. I have added notice on the [README](https://github.com/xuri/excelize#introduction) and [documentation website](https://xuri.me/excelize/en/base/installation.html#install) for this, and I've closed this. If you have any questions, please let me know, and reopen this anytime.

谢谢您的议题，在您创建新的议题之前，请确保不存在任何重复的议题。这与已有议题重复，请参考 #1465, #1595, #1603, #1608, #1614, #1619, #1620, #1621, #1623, #1633, #1637, #1641, #1642, #1648, #1651, #1652, #1656, #1657, #1660, #1663, #1670, #1686, #1734 和 #1752。Go 1.21.0 对 encoding/xml 标准库做了不兼容的更改，我已将该议题反馈至 Go 团队，并提交了修复补丁 golang/go#61881，已在 Go 1.21.1 中修复。请使用 Go 1.20.12 以及更早版本，或升级 1.21.1 及以上版本。我已经为此在[自述文档](https://github.com/xuri/excelize/blob/master/README_zh.md#简介)和[文档网站](https://xuri.me/excelize/zh-hans/base/installation.html#install)上添加了通知，因此我将关闭此议题。如果您有任何疑问，请告诉我，并可随时重新打开。

**Comment by xuri** [comment]
Thanks for your issue. If you are reporting a new issue, make sure that we do not have any duplicates already exist. If it does not work with Go 1.21.0, please reference the issues #1465, #1595, #1603, #1608, #1614, #1619, #1620, #1621, #1623, #1633, #1637, #1641, #1642, #1648, #1651, #1652, #1656, #1657, #1660, #1663, #1670, #1686, #1734, and #1752. There are some incompatible changes in the Go 1.21.0 encoding/xml library. I have given feedback to the Go team and created a patch for it (golang/go#61881), and it has been fixed on Go 1.21.1. Please using the Go 1.20.12 and previous Go released version or upgrade to Go 1.21.1 and later. I have added notice on the [README](https://github.com/xuri/excelize#introduction) and [documentation website](https://xuri.me/excelize/en/base/installation.html#install) for this, and I've closed this. If you have any questions, please let me know, and reopen this anytime.
